Experimental/Asteroids-CMB-NGO-Sample/Assets/Scenes/SmallDemos/Scripts/InSceneSerializationTest.cs

```diff
@@ -0,0 +1,149 @@
+using System;
```

Comment:
Could this best serve inside testproject or the internal project? It's unclear what to do inside the `SerializationValidation` scene, and if it's a validator of features, and not necessarily a use-case it might be best kept internally? Thoughts?